### ISSUE TITLE
Oberon export: Included global declarations are repeated locally by subroutines

### ISSUE THREAD

**Issue by codemanyak** [fesch/Structorizer.Desktop#776]
If several subroutine diagrams involved in a code export import declarations meant to be global from a common includable diagram then the declarations are (unduly) repeated locally in the subroutines, which is contra-productive:
Example:
![grafik](https://user-images.githubusercontent.com/15326471/68717831-0bde8b00-05a8-11ea-87ec-9ee0fb40f14b.png)

![grafik](https://user-images.githubusercontent.com/15326471/68717870-23b60f00-05a8-11ea-9415-71d1b3c1cd0c.png)

This yields the following defective export:
```oberon
MODULE HelloWorld-EVA_Principle;
(*
 * Every Program has principally the base-structure :
 * I nput (E ingabe)
 * P rocessing (V erarbeitung)
 * O utput (A usgabe)
 *)
(* Generated by Structorizer 3.30-02 *)
CONST
  helloworldstring = 'Hello World !';


VAR
  (* TODO: check and accomplish variable declarations *)
  inputstring: ARRAY 100 OF CHAR;
  outputstring: ARRAY 100 OF CHAR;

PROCEDURE I_nput*();
(*
 * The declared Input-Variables get their values.
 *)

VAR
  (* TODO: check and accomplish variable declarations *)
  inputstring: ARRAY 100 OF CHAR;
BEGIN
  inputstring := helloworldstring;
END I_nput;

PROCEDURE O_utput*();
(*
 * The computed values of the Output-Variables are put outside.
 *)

BEGIN
  Out.String(outputstring);
  Out.Ln;
END O_utput;

PROCEDURE P_rocessing*();
(*
 * The Algorithm, which computes the declared Output-Variables from the
declared Input-Variables.
 *)

VAR
  (* TODO: check and accomplish variable declarations *)
  outputstring: ARRAY 100 OF CHAR;
BEGIN
  outputstring := inputstring;
END P_rocessing;

BEGIN
  I_nput();
  P_rocessing();
  O_utput();
END HelloWorld-EVA_Principle.
```

**Comment by codemanyak** [comment]
First of all, it is only the mere declarations from the included diagrams that are inserted in diagrams assigning values to them. In analogy to #773, a special treatment for mere declarations is necessary here.

**Comment by codemanyak** [comment]
Fix is available on bugfix branch (filed for version 3.30-03).
